metapool/tests/test_sample_sheet.py

```diff
@@ -423,16 +425,17 @@ def test_merge_bioinformatics(self):
             columns=['Sample_Project', 'QiitaID', 'BarcodesAreRC',
                      'ForwardAdapter', 'ReverseAdapter', 'HumanFiltering',
                      'library_construction_protocol',
-                     'experiment_design_description'],
+                     'experiment_design_description', 'contains_replicates'],
             data=[
                 ['Koening_ITS_101', '101', 'False', 'GATACA', 'CATCAT',
-                 'False', 'Knight Lab Kapa HP', 'Eqiiperiment'],
+                 'False', 'Knight Lab Kapa HP', 'Eqiiperiment', False],
```

Comment:
Do we need to add a tests with True?

Reply:
Saw that one of the input files has a column with `contains_replicates == True` so maybe the test already exists, leaving here to confirm.

Reply:
The changes to the current tests do an adequate job of testing to make sure that the current error doesn't return. Of course more tests is never a bad thing especially with new functionality like replicates, but I think it's better to make that a separate issue instead of piggybacking additional tests onto a bug fix. I think it's better to get this merged and out to the wet-lab as soon as possible.